apps/aesophia/src/aeso_ast_to_icode.erl

```diff
@@ -703,11 +703,8 @@ builtin_function(abort) ->
     A = fun(X) -> aeb_opcodes:mnemonic(X) end,
     {{builtin, abort}, [private],
      [{"s", string}],
-     %% length, address, REVERT
-     {inline_asm, [A(?DUP1),
-                   A(?MLOAD),A(?SWAP1),         %Load size and swap
-                   A(?PUSH1),32,A(?ADD),        %Load address of string binary
```

Comment:
Trying to understand whether change is improvement:
Maybe this line (before change) was not handling [zero-padded strings](https://github.com/aeternity/protocol/blame/d3126f7ec91067bd10a35c8c02004645b51c95c0/contracts/sophia.md#L925)?

Reply:
The improvement is that it uses fewer byte code instructions. Nothing else.